### ISSUE TITLE
Accessibility - Cannot set aria-label on quickreplies carousel's buttons

### ISSUE THREAD

**Issue by correi-f** [theopenconversationkit/tock-react-kit#104]
User cannot set the attribute aria-label on buttons of the QR's carousel. 
In this example, we can do it for the card's carousel but not for the Inline QR's carousel.
![image](https://user-images.githubusercontent.com/2005914/176701104-9b6825cd-b864-462f-908c-22ebd4629036.png)
